tests/helper_functions.py

```diff
@@ -211,6 +212,19 @@ def pyfunc_serve_and_score_model(
     activity_polling_timeout_seconds=500,
     extra_args=None,
     stdout=sys.stdout,
+):
+    with pyfunc_scoring_endpoint(
+        model_uri,
+        extra_args=extra_args,
+        activity_polling_timeout_seconds=activity_polling_timeout_seconds,
+        stdout=stdout,
+    ) as endpoint:
+        return endpoint.invoke(data, content_type)
+
+
+@contextmanager
+def pyfunc_scoring_endpoint(
+    model_uri, activity_polling_timeout_seconds=500, extra_args=None, stdout=sys.stdout
```

Comment:
is it possible to default it to `[]`?

Reply:
ruff B006 doesn't allow using a mutable object as a default value:

https://docs.astral.sh/ruff/rules/mutable-argument-default/#example

Reply:
Oh interesting, I guess that makes sense